safesql/safesql.go

```diff
@@ -27,12 +30,18 @@
 // the uncheckedconversions package can be used to assert that those strings are under the programmer control.
 // Note that unchecked conversions should be very limited, ideally never used, as they pose a security risk.
 //
-// Note on missing documentation
+// Note on API documentation.
 //
 // For documentation on methods and types that wrap the standard ones please refer to the stdlib package doc instead, as
 // all the types exported by this package are tiny wrappers around the standard ones and thus follow their behavior.
 // The only relevant difference is that functions accept TrustedSQLString instances instead of plain "strings" and that some
 // dangerous methods have been removed.
+//
+// Explainer
+//
+// This package wraps the sql package and all methods that would normally take a `string` take a TrustedSQLString instead.
```

Comment:
I have often got readability comments about not using backticks as they don't render in any special way in GoDoc.